src/main/java/bio/terra/testrunner/runner/config/ServerSpecification.java

```diff
@@ -111,6 +111,10 @@ public static ServerSpecification fromJSONFile(String resourceFileName) throws E
    */
   public void validate() {
     if (!skipKubernetes) {
+      if (cluster == null) {
+        throw new IllegalArgumentException(
+                "Cluster Specification must be defined");
+      }
```

Comment:
I think the changes to this file are from a different PR -- can you remove them from here please to avoid confusion?

Reply:
These changes have been removed.